OxfordBodleian/Juel-Jensen/BDLaethf25.xml

```diff
@@ -153,21 +586,226 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                        <q xml:lang="gez">
+                                            <hi rend="rubric">
+                                                ሰሐል፡ ሰሐል፡ ሰሐል፡ ጸጸጸጸጸጸጸጸጸጸጸጸጸጸጸጸጸጸጸጸጸጸጸጸ<del rend="strikethrough">ጸ</del>እዴየ፡ 
+                                                ሰሐል፡ ወየማንየ፡ አሕመላምል፡ ለጺሒፈ፡ ብርዕ፡ ወቀለም፡ ሊተ፡ ለገብርከ፡ አፍጌት፡ 
+                                                ወርኃ፡ ደቡብ፡ ወወርኃ፡ ደመና፡ <del rend="encircled strikethrough">አክሄል፡ ድሜሄላዊ፡</del> 
+                                                መንጥሔል፡ ድሜሄላዊ፡ በከመ፡ የሐውሩ፡ ፀሐይ፡ ወወርኅ፡ በፍጥረተ፡ 
+                                                ትእዛዝከ፡ ከማሆሙ፡ አሩፅ፡ የማነ፡ እዴየ፡ ለጋ፡ በስመ፡ 
+                                                <del rend="strikethrough">ለለ</del>ሎኤ፡ ኢላኤ፡ ኢላሄ፡ አሩፅ፡ እዴየ፡ 
+                                                ለገቢ<pb n="78r"/>ረ፡ ኵሉ፡ ተግባር፡ በሩፋኤል፡ ብርናዊ፡ 
+                                                በክሳብኤል፡ ዘ<del rend="strikethrough">ፈ</del>ያረውፃ፡ ለፀሐይ፡ 
+                                                ከማሁ፡ አሩፅ፡ እዴየ፡ ከመ፡ ይጠንቅቅ፡ ለአኂዘ፡ ብርዕ፡ ወእጽሐፍ፡ 
+                                                እበያቲከ፡ ዘእንበለ፡ ሐኬት፡ ኦእግዚእየ፡ ኢየሱስ፡ ክርስቶስ፡ 
+                                                አስተ<surplus>፡</surplus> ፋጥን<supplied reason="undefined">፡</supplied> 
+                                                ወ<add place="above">አ</add>ስተ<surplus>፡</surplus> 
+                                                ቃንዕ፡ ለገብርከ፡ በቀይ፡ ቀለም፡ <del rend="strikethrough">ወ</del>በወረቀት፡ 
+                                                ጸሐፍዕስር<supplied reason="undefined">፡</supplied>
+                                            </hi>
+                                        </q>.
+                                </item>
+
+                                <item xml:id="e1">
+                                    <locus target="#1r"/>
+                                    <desc type="OwnershipNote"> 
+                                        </desc>
+                                        <q xml:lang="en">
+                                            MS Aeth. J-J. 27<lb/>
+                                            Widdasie Maryan
+                                        </q>.
+                                </item>
+                                
+                                <item xml:id="e2">
+                                    <locus target="#1r"/>
+                                    <desc type="Unclear">The "N" has been written mirrored.
+                                    </desc>
+                                        <q xml:lang="la">
+                                            INRI
+                                        </q>.
+                                </item>
+
+                                <item xml:id="e3">
+                                    <locus target="#1r"/>
+                                    <desc type="Unclear"> 
+                                    </desc>
+                                        <q xml:lang="gez">
```

Comment:
unfortunately, this encoding will lead to a visualisation problem, because the typographical requirements for text marked up as geez differ from those for Latin text. So despite the inconsistency, it would make our life later easier if this were encoded as latin or english